spirv_msl.cpp

```diff
@@ -561,6 +576,32 @@ void CompilerMSL::build_implicit_builtins()
 			set_name(var_id, "spvDispatchBase");
 			builtin_dispatch_base_id = var_id;
 		}
+
+		if (need_sample_mask && !does_shader_write_sample_mask)
+		{
+			uint32_t offset = ir.increase_bound_by(3);
+			uint32_t type_id = offset;
+			uint32_t var_id = offset + 2;
+
+			// Create gl_SampleMask.
+			SPIRType uint_type;
```

Comment:
This should use the get_uint_type_id() stuff that was merged a few hours ago.

Reply:
Done. I hope that's the correct way to merge stuff.

Reply:
I think you need a rebase.

Reply:
Also, there was an update of glslang/spirv-tools references in the interim as well, so beware that you probably need to rebuild that locally to create the reference test files.

Reply:
Should be fine now.